static/js/chart/draw_choropleth.ts

```diff
@@ -81,7 +81,9 @@ function drawChoropleth(
   canClick: boolean,
   getRedirectLink: (geoDcid: GeoJsonFeatureProperties) => string,
   getTooltipHtml: (place: NamedPlace) => string,
-  zoomDcid?: string
+  zoomDcid?: string,
```

Comment:
please document this growing list of parameters :)

Reply:
done